nilearn/signal.py

```diff
@@ -417,6 +417,13 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
         nilearn.image.clean_img
     """
 
+    if isinstance(low_pass, bool):
+        raise TypeError("low pass must be float or None but you provided "
+                        "low_pass='{0}'".format(low_pass))
```

Comment:
@KamalakerDadi I just corrected the message as @lesteve suggested. Look at his outdated comment above with single quotes. I don't have a preference on this. which one is better practice? If you want I will change it again.